es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number}${street}`;
+
+    console.log('LearningFuze:',address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:',bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit} ${item}`;
```

Comment:
This isn't an exact conversion:

---

![image](https://user-images.githubusercontent.com/16143771/127085004-6c993467-8928-4038-a6bf-68c5507f752a.png)

---

![image](https://user-images.githubusercontent.com/16143771/127084090-1028ca03-532d-43bf-b17d-3feb0d08ec59.png)